drivers/utimer/include/utimer.h

```diff
@@ -0,0 +1,507 @@
+/* SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright (C) 2024 Alif Semiconductor.
```

Comment:
```suggestion
 * Copyright (C) 2025 Alif Semiconductor.
```
Same comment applies to all the other files too.